content/_partials/toptoolbar.html.haml

```diff
@@ -13,71 +13,67 @@
 
         %li.nav-item.dropdown
           .nav-link.dropdown-toggle{"aria-expanded" => "false", "aria-haspopup" => "true", "data-toggle" => "dropdown", :role => "button"}
-            Documentation
+            Docs
 
           .dropdown-menu
             %a.dropdown-item{:href => expand_link('doc')}
               Use Jenkins
             %a.dropdown-item{:href => expand_link('doc/developer')}
               Extend Jenkins
+            %span.dropdown-item
+              %strong
+                Use-cases
+            - site.solutions.keys.sort.each do |key|
+              - link = expand_link("solutions/#{key}")
+              %a.dropdown-item.feature{:href => link, :class => active_link?(key.to_s)}
+                = '&nbsp;-&nbsp;' + site.solutions[key].usecase
+
 
         %li.nav-item{:class => active_link?('plugins')}
           %a.nav-link{:href => 'https://plugins.jenkins.io/'}
             Plugins
 
         %li.nav-item.dropdown
           .nav-link.dropdown-toggle{"aria-expanded" => "false", "aria-haspopup" => "true", "data-toggle" => "dropdown", :role => "button"}
-            Use-cases
-          .dropdown-menu
-            - site.solutions.keys.sort.each do |key|
-              - link = expand_link("solutions/#{key}")
-              %a.dropdown-item.feature{:href => link, :class => active_link?(key.to_s)}
-                = site.solutions[key].usecase
-
-        %li.nav-item{:class => active_link?('participate')}
-          %a.nav-link{:href => expand_link("participate")}
-            Participate
+            Community
 
-        %li.dropdown.nav-item
-          .nav-link.dropdown-toggle{"aria-expanded" => "false", "aria-haspopup" => "true", "data-toggle" => "dropdown", :role => "button"} Sub-projects
           .dropdown-menu
-            %a.dropdown-item.feature{:href => expand_link('projects/')}
+            %a.dropdown-item.feature{:href => expand_link("participate")}
               Overview
```

Comment:
Not sure I'd call `/participate` an _overview_ given the rest of the items.

Reply:
Alternative suggestion? It has a lot of overlap.